civihr_default_theme/scss/components/_container.scss

```diff
@@ -0,0 +1,78 @@
+/**
+* container for report
+**/
+
+.chr-separator {
+  background-color: $light-gray;
+  height: 2px;
+}
+
+.chr-container {
+  background-color: $gray-light;
```

Comment:
Why do you need to specify the `background-color`? I don't think it's necessary, the component will take the bg color of its parent